qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddUserTest.java

```diff
@@ -38,26 +34,9 @@ public void addUserTest() {
             manageUserPage.removeUser("super_nurse");
         } else {
             AddEditUserPage addEditUserPage = manageUserPage.clickOnAddUser();
-            addEditUserPage.createNewPerson();
-
-            addEditUserPage.saveUser();
-            List<String> validationErrors = addEditUserPage.getValidationErrors();
-            assertTrue(validationErrors.contains("You must define at least one name"));
-            assertTrue(validationErrors.contains("Cannot be empty or null"));
-            assertFalse(addEditUserPage.isDataCorrect(validationErrors));
-
-            addEditUserPage.enterGivenFamily("Super", "Nurse");
```

Comment:
any reason for removing the assertions?

Reply:
> any reason for removing the assertions?
@kdaud initially the test was made to rely on [addEditUserPage](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddEditUserPage.java), however, It should rely on [CreateNewUserPage](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/CreateNewUserPage.java). Now given the coupling of methods in the createNewUserPage, there's no room for errors.  You could advise me on the best approach to take @kdaud

Reply:
any reason for switching the test from relying on `AddEditUserPage` to `CreateNewUserPage`?

Reply:
@kdaud the actual page that is meant to implement it is the CreateNewUserPage, you follow their URLs

Reply:
> the actual page that is meant to implement it is the CreateNewUserPage.

@Irenyak1 could you tell me more?

Reply:
There are two ways of creating user account and the test is using the second approach [here](https://talk.openmrs.org/t/creating-user-accounts-in-reference-application/34019). You need to investigate why the test is failing the [validation declaration](https://github.com/openmrs/openmrs-contrib-qaframework/blob/0d34bada84b09c2d83cc40043a8a3bb392af20b4/qaframework-legacy-tests/src/test/java/org/openmrs/contrib/qaframework/legacy/AddUserTest.java#L44).

Reply:
@kdaud  I am looking into why the validations are failing. I am still getting these [errors ](https://pastebin.com/v0KMWdLb). When the test clicks save user and the validation errors are displayed, it looks like the session is killed and the test can not proceed after that. Any one to guide me over this blocker @kdaud , @sherrif10 , @dkayiwa @ibacher

Reply:
> When the test clicks save user and the validation errors are displayed, it looks like the session is killed and the test can not proceed after that.

@Irenyak1 investigate why the test run is killed  at that point.

Reply:
@kdaud did you take time to look into the error logs?

Reply:
> @kdaud did you take time to look into the error logs?

yes. That is why I made the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/299#discussion_r776959967) !

Reply:
> @kdaud did you take time to look into the error logs?

yes. That is why I made the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/299#discussion_r776959967) !

Reply:
> That is why I made the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/299#discussion_r776959967) !

What were your findings ?

Reply:
> What were your findings ?

haven't investigated since you're looking into the test.